lib/packwerk/cli.rb

```diff
@@ -70,8 +73,6 @@ def execute_command(args)
       end
     end
 
-    private
```

Comment:
Why was this removed?

Reply:
Testing purposes. The private keyword is added back later on in the file for methods that don't get called from `execute_command`